tests/integration-tests/configs/common/common.yaml

```diff
@@ -483,6 +483,13 @@ storage:
         instances: {{ common.INSTANCES_DEFAULT_X86 }}
         oss: ["alinux2"]
         schedulers: ["slurm"]
+  # Ephemeral test requires instance type with instance store
+  test_ephemeral.py::test_head_node_stop:
+    dimensions:
+      - regions: ["us-east-1"]
+        instances: ["m5d.xlarge", "h1.2xlarge"]
+        oss: ["alinux2"]
+        schedulers: ["slurm"]
```

Comment:
If the logic is scheduler independent I'd test awsbatch in place of slurm to save some route53 calls (aka throttling)

Reply:
ok, done